flareon/src/response.rs

```diff
@@ -86,6 +130,33 @@ mod tests {
         );
     }
 
+    #[test]
+    #[cfg(feature = "json")]
+    fn response_new_json() {
+        #[derive(serde::Serialize)]
+        struct MyData {
+            hello: String,
+        }
+
+        let data = MyData {
+            hello: String::from("world"),
+        };
+        let response = Response::new_json(StatusCode::OK, &data).unwrap();
+        assert_eq!(response.status(), StatusCode::OK);
+        assert_eq!(
+            response.headers().get(http::header::CONTENT_TYPE).unwrap(),
+            JSON_CONTENT_TYPE
+        );
+        match &response.body().inner {
+            BodyInner::Fixed(fixed) => {
+                assert_eq!(fixed, "{\"hello\":\"world\"}");
```

Comment:
Nit: in another place, we're using `r#"{"hello":"world"}"#` for that case. It would be nice to represent it in one way

Reply:
Fair